packages/searchable-dropdown/src/dropdown/element.ts

```diff
@@ -131,12 +131,18 @@ export class SearchableDropdownElement
   @query('fwc-list')
   listElement: ListElement | undefined;
 
+  updated(props: Map<string, string | null | undefined>) {
+    if (props.has('selectedId')) {
+      this.controller.initialItemsMutation();
+    }
+  }
+
   /* Build fwc-list-items */
   protected buildListItem(item: SearchableDropdownResultItem): HTMLTemplateResult {
     this.controller._listItems.push(item.id);
     const itemClasses = {
       'list-item': true,
-      'item-selected': !!item.isSelected,
+      'item-selected': !!item.isSelected || !!this.controller._selectedItems.find((si) => si.id === item.id),
```

Comment:
previously it looks like `isSelected` is set by `mutateResult`. That method is modified now.